source/mrmeshpy/MRPythonMeshExposing.cpp

```diff
@@ -548,6 +549,15 @@ MR_ADD_PYTHON_CUSTOM_DEF( mrmeshpy, SimpleFunctions, [] ( pybind11::module_& m )
         "finds all pairs of colliding triangles from two meshes or two mesh regions\n"
         "\trigidB2A - rigid transformation from B-mesh space to A mesh space, nullptr considered as identity transformation\n"
         "\tfirstIntersectionOnly - if true then the function returns at most one pair of intersecting triangles and returns faster" );
+
+
+    m.def( "makeDegenerateBandAroundRegion",
+        [=] ( Mesh& mesh, const FaceBitSet& faces ) { makeDegenerateBandAroundRegion( mesh, faces ); },
```

Comment:
why not just `&MR::makeDegenerateBandAroundRegion`

Reply:
makeDegenerateBandAroundRegion has incompatible signature due to third argument `MakeDegenerateBandAroundRegionParams = {}`